src/file-utils.ts

```diff
@@ -83,3 +84,21 @@ export const readMachOHeader = async (path: string) => {
   });
   return Buffer.concat(chunks);
 };
+
+export const fsMove = async (oldPath: string, newPath: string) => {
```

Comment:
Needing to port code like this makes me wonder if `fs-extra` is worth keeping around just for the extra utility since we're still using `graceful-fs` for the most part 🫨